indico/MaKaC/webinterface/wcomponents.py

```diff
@@ -51,6 +51,7 @@
 from indico.modules.api import settings as api_settings
 from indico.modules.events.layout import layout_settings, theme_settings
 from indico.modules.events.util import preload_events
+from indico.modules.legal import settings as legal_settings
```

Comment:
we stopped calling the proxies just `settings` in more recent code so i'd rename it to `legal_settings` and get rid of the `as` part here